eva_submission/submission_qc_checks.py

```diff
@@ -0,0 +1,216 @@
+                if variants_skipped_line:
+                    logger.error(f'Some of the variants were skipped while accessioning. '
+                                 f'The following line from logs describes the total variants that were skipped.'
+                                 f'\n{variants_skipped_line}')
+            else:
+                logger.error(f'No accessioning log file could be found for study {project_accession}')
+
+    def check_if_variant_load_completed_successfully(self, project_accession, vcf_files, path_to_data_dir):
+        for file in vcf_files:
+            pipeline_log_files = glob.glob(f"{path_to_data_dir}/00_logs/pipeline.*{file}*.log")
+            if pipeline_log_files:
+                # check if variant load job completed successfully
+                if self.check_if_job_completed_successfully(pipeline_log_files[0]):
+                    logger.info(f'Variant load stage completed successfully for study {project_accession} '
+                                f'and file {os.path.basename(pipeline_log_files[0])}')
+                else:
+                    logger.error(f'Variant load stage failed for study {project_accession} '
+                                 f'and file {os.path.basename(pipeline_log_files[0])}')
+
+                    self.check_for_errors_in_case_of_job_failure(pipeline_log_files[0])
+            else:
+                logger.error(f'No pipeline log file could be found for study {project_accession}')
+
+    def check_if_browsable_files_entered_correctly_in_db(self, vcf_files, profile, private_config_xml_file,
+                                                         project_accession):
+        browsable_files_from_db = self.get_browsable_files_for_study(profile, private_config_xml_file,
+                                                                     project_accession)
+        if set(vcf_files) - set(browsable_files_from_db):
+            logger.error(f"There are some VCF files missing in db. "
+                         f"Missing Files : {set(vcf_files) - set(browsable_files_from_db)}")
+        else:
+            logger.info(f"Browsable Files entered correctly in DB. Browsable files : {vcf_files}")
+
+    def run_qc_checks_for_submission(self):
+        profile = cfg['maven']['environment']
+        private_config_xml_file = cfg['maven']['settings_file']
+        project_accession = self.eload_cfg.query('brokering', 'ena', 'PROJECT')
+        path_to_data_dir = Path(cfg['projects_dir'], project_accession)
+        taxonomy = self.eload_cfg.query('submission', 'taxonomy_id')
+        analyses = self.eload_cfg.query('brokering', 'analyses')
+
+        vcf_files = []
+        for analysis_data in analyses.values():
+            for v_files in analysis_data['vcf_files'].values():
+                vcf_files.append(os.path.basename(v_files['output_vcf_file']))
+
+        logger.info(f'----------------------------Check Browsable Files Entered Correctly-----------------------------')
```

Comment:
I think this would be more user-friendly if it were a formatted report like the [validation report](https://github.com/EBIvariation/eva-submission/blob/de1a677f3d7801f913355aa1ea996a82c7021091/eva_submission/eload_validation.py#L621) rather than a series of log messages.

Reply:
done